frontend/packages/dev-console/integration-tests/features/pipelines/pipelines-runs.feature

```diff
@@ -96,90 +96,113 @@ Scenario Outline: Pipeline Run Details page for a pipeline with resource : P-06-
     And Pipeline Resources field will be displayed
 
 Examples:
-| pipeline_name             |
-| pipe-task-with-resoruce-4 |
+| pipeline_name          |
+| pipeline-with-resoruce |
 
 
 @regression, @smoke
 Scenario Outline: Filter the pipeline runs based on status : P-06-TC07
     Given pipeline "<pipeline_name>" is executed for 3 times
     When user filters the pipeline runs of pipeline "<pipeline_name>" based on the "<status>"
-    Then user able to see the pipelineruns with "<status>"
+    Then user is able to see the pipelineruns with "<status>"
 
 Examples:
-| pipeline_name              | status    |
-| pipe-task-without-resoruce | Succeeded |
+| pipeline_name             | status    |
+| pipeline-without-resoruce | Succeeded |
 
 
 @regression, @smoke
-Scenario Outline: Start the pipeline from Pipeline Details page : P-04-TC04
-    Given pipeline "<pipeline_name>" is available in pipelines page
+Scenario: Start the pipeline from Pipeline Details page : P-04-TC04
+    Given pipeline "pipeline-one" is available in pipelines page
     When user selects "Start" option from pipeline Details Actions menu
     Then user will be redirected to Pipeline Run Details page
 
-Examples:
-| pipeline_name             |
-| pipe-task-with-resoruce-8 |
-
 
 @regression, @manual
 Scenario Outline: Download the logs from Pipeline Details page : P-04-TC05
-    Given user is at the Pipeline Details page
+    Given pipeline "pipeline-one" is available in pipelines page
+    When user selects "Start" option from kebab menu in pipelines page 
+    And user navigates to pipelineRun logs tab
+    And user clicks on Download button
+    Then user is able to see the downloaded file
 
 
-@regression
-Scenario: kebab menu options in pipelines page : P-04-TC07
-    Given user is at pipelines page
+@regression, @manual
+Scenario Outline: Download the logs from Pipeline Details page : P-05-TC06
+    Given pipeline run is displayed for "pipe-task-with-resoruce" with resource
+    When user navigates to pipelineRun logs tab
+    And user clicks on Download button
+    Then user is able to see the downloaded file
+    And logs should be displayed as expected
```

Comment:
as expected can be replaced by `with tasks as steps with details of execution`

Reply:
updated as "logs contains tasks with details of execution"